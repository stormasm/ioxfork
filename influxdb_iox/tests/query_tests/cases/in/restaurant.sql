-- IOX_SETUP: TwoMeasurementsUnsignedType

-- IOX_COMPARE: sorted
SELECT town, count from restaurant;

-- IOX_COMPARE: sorted
SELECT town, count from school;
