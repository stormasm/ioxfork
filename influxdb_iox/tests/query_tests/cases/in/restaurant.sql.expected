-- Test Setup: TwoMeasurementsUnsignedType
-- SQL: SELECT town, count from restaurant;
-- Results After Sorting
+---------+-------+
| town    | count |
+---------+-------+
| andover | 40000 |
| reading | 632   |
+---------+-------+
-- SQL: SELECT town, count from school;
-- Results After Sorting
+---------+-------+
| town    | count |
+---------+-------+
| andover | 25    |
| reading | 17    |
+---------+-------+